src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -808,6 +808,261 @@ public void isSentToMultiSig_whenNonStandardErpRedeemScript_shouldReturnTrue() {
         assertScriptIsMultiSig(nonStandardErpRedeemScript);
     }
 
+    @Test
+    public void getSigsPrefixCount_whenP2PKScript_shouldReturnZero() {
```

Comment:
Should we clarify in these test names that we are referring to output scripts maybe?